modules/os2forms_digital_post/src/Drush/Commands/DigitalPostTestCommands.php

```diff
@@ -98,21 +116,40 @@ public function send(
       throw new InvalidArgumentException(sprintf('Invalid type: %s. Must be one of %s.', $quote($type), implode(', ', array_map($quote, SF1601::TYPES))));
     }
 
+    $meMoVersion = $options['memo-version'];
+    if ($meMoVersion) {
+      $meMoVersion = (float) $meMoVersion;
+      $allowedValues = [SF1601::MEMO_1_1, SF1601::MEMO_1_2];
+      if (!in_array($meMoVersion, $allowedValues, TRUE)) {
+        $quote = static fn($value) => var_export($value, TRUE);
+        throw new InvalidArgumentException(sprintf(
+          'Invalid MeMo version: %s. Must be one of %s.',
+          $quote($meMoVersion),
+          implode(', ', array_map($quote, $allowedValues))
+        ));
+      }
+    }
+
     $io->section('Digital post');
     $io->definitionList(
       ['Type' => $type],
       ['Subject' => $subject],
-      ['Message' => $message]
+      ['Message' => $message],
+      ['Document' => sprintf('%s (%s)', $document->filename, $document->mimeType)],
+      ['MeMo version' => $meMoVersion ?? '–'],
     );
 
+    $actions = array_map($this->buildAction(...), $options['action']);
+
     foreach ($recipients as $recipient) {
       try {
-        $io->writeln(sprintf('Recipient: %s', $recipient));
         $recipientLookupResult = $this->digitalPostHelper->lookupRecipient($recipient);
-        $actions = [];
 
         $meMoMessage = $this->digitalPostHelper->getMeMoHelper()->buildMessage($recipientLookupResult, $senderLabel,
           $messageLabel, $document, $actions);
+        if ($meMoVersion) {
+          $meMoMessage->setMemoVersion($meMoVersion);
+        }
```

Comment:
is this possible that we don't have `$meMoVersion` at this point?

the function doc says there is a default value to be provided. When is that happening? Code/documentation can be bit more clear on that.